WordPress/src/main/res/values-ar/strings.xml

```diff
@@ -31,86 +31,86 @@ Language: ar
-    <string name="gutenberg_native_double_tap_and_hold_to_edit">\"انقر نقرًا مزدوجًا مع الاستمرار للتحرير\"</string>
-    <string name="gutenberg_native_current_value_is_s">\"القيمة الحالية هي %s\"</string>
-    <string name="gutenberg_native_content">\"المحتوى…\"</string>
-    <string name="gutenberg_native_clear_all_settings">\"مسح كافة الإعدادات\"</string>
-    <string name="gutenberg_native_choose_from_device">\"الاختيار من جهاز\"</string>
-    <string name="gutenberg_native_annotations_sidebar">\"الشريط الجانبي للتعليقات التوضيحية\"</string>
-    <string name="gutenberg_native_an_unknown_error_occurred_please_try_again">\"حدث خطأ غير معروف. يُرجى المحاولة مرة أخرى.\"</string>
-    <string name="gutenberg_native_alt_text">\"النص البديل\"</string>
-    <string name="gutenberg_native_add_video">\"إضافة فيديو\"</string>
-    <string name="gutenberg_native_add_url">\"إضافة عنوان URL\"</string>
-    <string name="gutenberg_native_add_link_text">\"إضافة نص رابط\"</string>
-    <string name="gutenberg_native_add_image_or_video">\"إضافة صورة أو فيديو\"</string>
-    <string name="gutenberg_native_add_image">\"إضافة صورة\"</string>
-    <string name="gutenberg_native_add_block_here">\"إضافة مكوِّن هنا\"</string>
-    <string name="gutenberg_native_add_annotation">\"إضافة تعليق توضيحي\"</string>
-    <string name="gutenberg_native_add_a_description">\"إضافة وصف\"</string>
+    <string name="gutenberg_native_wordpress_media_library">مكتبة وسائط ووردبريس</string>
+    <string name="gutenberg_native_we_are_working_hard_to_add_more_blocks_with_each_release_in_the_m">نعمل جاهدين لإضافة المزيد من المكوِّنات لكل إصدار. في الوقت الحالي، يمكنك أيضًا تحرير هذا المقال على الويب.</string>
+    <string name="gutenberg_native_unsupported">غير مدعوم</string>
+    <string name="gutenberg_native_ungroup">إلغاء التجميع</string>
+    <string name="gutenberg_native_translate">ترجمة</string>
+    <string name="gutenberg_native_title">العنوان:</string>
+    <string name="gutenberg_native_tap_to_hide_the_keyboard">انقر لإخفاء لوحة المفاتيح</string>
+    <string name="gutenberg_native_tap_here_to_show_help">انقر هنا لإظهار المساعدة</string>
+    <string name="gutenberg_native_take_a_video">تسجيل فيديو</string>
+    <string name="gutenberg_native_take_a_photo_or_video">التقاط صورة أو فيديو</string>
+    <string name="gutenberg_native_take_a_photo">التقاط صورة</string>
+    <string name="gutenberg_native_start_writing">بدء الكتابة…</string>
+    <string name="gutenberg_native_size">الحجم</string>
+    <string name="gutenberg_native_sidebar_title_plugin">المكون الإضافي لعنوان الشريط الجانبي</string>
+    <string name="gutenberg_native_show_section">إظهار القسم</string>
+    <string name="gutenberg_native_s_isn_t_yet_supported_on_wordpress_for_ios">\'لم يتم دعم \'%s\' بعد على ووردبريس لـ iOS</string>
+    <string name="gutenberg_native_s_isn_t_yet_supported_on_wordpress_for_android">\'لم يتم دعم \'%s\' بعد على ووردبريس لـ Android</string>
```

Comment:
The `%s` is correctly surrounded by a `\'`, it seems there is an extra `\'` at the string end (RTL).

Reply:
Right 👍